task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -123,18 +125,23 @@ spec:
         #!/usr/bin/env bash
         source /utils.sh
 
-        HACBS_ERROR_OUTPUT=$(make_result_json -r "ERROR")
         if [ -f /tekton/home/clamscan-result.json ];
         then
           cat /tekton/home/clamscan-result.json
           INFECTED_FILES=$(jq -r '.infected_files' /tekton/home/clamscan-result.json || true )
           if [ -z "${INFECTED_FILES}" ]; then
             echo "Failed to get number of infected files"
+            note="Task $(context.task.name) meets error: Failed to get number of infected files from /tekton/home/clamscan-result.json, please check the tekton task log"
           else
             if [[ "${INFECTED_FILES}" -gt 0 ]]; then RES="FAILURE"; else RES="SUCCESS"; fi
-            HACBS_TEST_OUTPUT=$(make_result_json -r "${RES}" -s 1 -f "${INFECTED_FILES}")
+            note="Task $(context.task.name) completes, please check RESULT for the antivirus scan result"
+            HACBS_TEST_OUTPUT=$(make_result_json -r "${RES}" -s 1 -f "${INFECTED_FILES}" -t $note)
           fi
+        else
+          note="Task $(context.task.name) meets error: File /tekton/home/clamscan-result.json doesn't exist, please check the tekton task log"
```

Comment:
I'm not sure if `meets error` is a proper english term. It should be `Task failed` or `Task encountered an error`

But I'm not a native speaker, feel free to correct me

Reply:
@MartinBasti can you get me a screen cap of this message in the UI? that'll help me to edit this.

Reply:
Currently we can see it in task log and EC team use it. The log is like this:
![image](https://user-images.githubusercontent.com/34546888/224249500-309bd7ab-d6ac-4411-a0df-b4e045bb8f4d.png)

Reply:
@jseseCCS We have to use `\` to escape ` in "Note", otherwise, the content will be considered as a command to execute.